.github/workflows/security.yml

```
@@ -0,0 +1,171 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Check file permissions
+        run: |
+          echo "🔍 Checking file permissions and integrity..."
+          
+          # Check for executable files that shouldn't be
+          find . -type f -executable -name "*.md" -o -name "*.json" -o -name "*.yml" -o -name "*.yaml" | while read file; do
+            echo "⚠️ Unexpected executable file: $file"
+          done
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)